README.md

```diff
@@ -6,6 +6,18 @@ D3 4.0 implementation of the Sankey plugin to visualize the flow between nodes i
 
 If you use NPM, `npm install d3-sankey`. Otherwise, download the [latest release](https://github.com/d3/d3-sankey/releases/latest).
 
+You can also load directly from unpkg.
+
+```javascript
+<script src="https://d3js.org/d3.v4.js"></script>
+<script src="https://unpkg.com/d3-sankey@0.4.1"></script>
```

Comment:
Or perhaps just the minor version for 0.x releases and the major version for 1.x+ releases.